lintreview/review.py

```diff
@@ -41,6 +41,50 @@ def __repr__(self):
             self.body)
 
 
+class IssueLabel(object):
+
+    OK_LABEL = 'No lint errors'
+
+    def __init__(self, label):
+        self.label = label
+
+    def remove(self, gh, pull_request_number):
+        try:
+            labels = gh.issues.labels.list_by_issue(pull_request_number)
+            if not any(self.label == label.name for label in labels):
+                return
+            log.debug("Removing issue label '%s'", self.label)
+            gh.issues.labels.remove_from_issue(pull_request_number, self.label)
+        except:
+            log.warn("Failed to remove label '%s'", self.label)
+
+    def publish(self, gh, pull_request_number):
+        # remove+add to show latest activity
+        self.remove(gh, pull_request_number)
+        log.debug("Publishing issue label '%s'", self.label)
+        try:
+            #gh.issues.labels.add_to_issue(pull_request_number, [self.label])
```

Comment:
`add_to_issue` is broken. I added a comment to clarify this. If it's ever fixed this line should replace the following work-around.